sonar-db/src/main/java/org/sonar/db/permission/PermissionDao.java

```diff
@@ -98,6 +117,21 @@ public int countGroups(DbSession session, String permission, @Nullable Long comp
   }
 
   /**
+   * ordered by group names
+   */
+  public List<String> selectGroupNamesByPermissionQuery(DbSession dbSession, PermissionQuery query) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document the parameter(s): dbSession, query [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUndocumentedApi)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this method return value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUndocumentedApi)